### ISSUE TITLE
Liblinphone.so.10 is so big

### ISSUE THREAD

**Issue by fengqz627** [BelledonneCommunications/linphone-desktop#686]

**Comment by KyronNextLevel** [comment]
Hello. Can you please list the command with which you compiled linphone-desktop and linphone-android please ?
Many things could lead to a big shared library.
Try disabling some features used by default that you don't need, and it would reduce the size.

Regards

**Comment by wmailn** [comment]
I compiled using the default configuration，I wonder where I can see which configurations I can turn off，thank you。

**Comment by Abduahad** [comment]
@wmailn @KyronNextLevel @fengqz627 
Can you share how to disable features?

**Comment by EugeneAsti** [comment]
yep! Same problem on Android. And I didn't figure out how can I change compile configuration to reduce app size. Using mobile version 5.3.63

**Comment by fengqz627** [comment]
some music files use too many sizes, for example ring file

--------------原始邮件--------------
发件人："EugeneAsti ***@***.***&gt;;
发送时间：2024年8月5日(星期一) 上午10:15
收件人："BelledonneCommunications/linphone-desktop" ***@***.***&gt;;
抄送："冯强周 ***@***.***&gt;;"Mention ***@***.***&gt;;
主题：Re: [BelledonneCommunications/linphone-desktop] Liblinphone.so.10 is so big (Issue #686)
-----------------------------------

 

 
yep! Same problem on Android. And I didn't figure out how can I change compile configuration to reduce app size. Using mobile version 5.3.63

**Comment by EugeneAsti** [comment]
The problem is not in music files. It is because of some internal libraries for various platforms.
